CARMAchem_GridComp/CARMAchem_Registry.rc

```diff
@@ -105,15 +108,28 @@ fraction_terpene_to_organic_carbon:     0.1
 # Point-wise sulfate source
 point_emissions_srcfilen_sulfate: /dev/null
 
-# Mie Table Information
+# Monochromatic (diagnostic) optical properties
   n_channels: 4
   n_moments:  0
   r_channels: 4.7e-7 5.5e-7 6.7e-7 8.7e-7
-  DU_OPTICS:  /discover/nobackup/pcolarco/fvInput/AeroCom/x/carma_optics_DU.v15.nbin=22.nc
-  SS_OPTICS:  /discover/nobackup/pcolarco/fvInput/AeroCom/x/carma_optics_SS.v3_3.nbin=22.nc
-  BC_OPTICS:  /discover/nobackup/pcolarco/fvInput/AeroCom/x/carma_optics_SU.v1.nbin=22.nc
-  SM_OPTICS:  /discover/nobackup/pcolarco/fvInput/AeroCom/x/carma_optics_SM.v1.nbin=22.nc
-  SU_OPTICS:  /discover/nobackup/pcolarco/fvInput/AeroCom/x/carma_optics_SU.v6.nbin=22.nc
+  filename_optical_properties_DU:  /discover/nobackup/pcolarco/fvInput/AeroCom/x/carma_optics_DU.v15.nbin=22.nc
```

Comment:
@pcolarco Just a query: Will you ever run this code at NAS? If so, we should work to get these files in a generic path like the opticsBands files below.